aiida/cmdline/commands/cmd_data/cmd_singlefile.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+###########################################################################
+# Copyright (c), The AiiDA team. All rights reserved.                     #
+# This file is part of the AiiDA code.                                    #
+#                                                                         #
+# The code is hosted on GitHub at https://github.com/aiidateam/aiida-core #
+# For further information on the license, see the LICENSE.txt file        #
+# For further information please visit http://www.aiida.net               #
+###########################################################################
+"""`verdi data singlefile` command."""
+from __future__ import division
+from __future__ import print_function
+from __future__ import absolute_import
+
+from aiida.cmdline.commands.cmd_data import verdi_data
+from aiida.cmdline.params import arguments, types
+from aiida.cmdline.utils import decorators, echo
+
+
+@verdi_data.group('singlefile')
+def singlefile():
+    """Work with SinglefileData nodes."""
+
+
+@singlefile.command('content')
+@arguments.DATA(type=types.DataParamType(sub_classes=('aiida.data:singlefile',)))
```

Comment:
Do we really want to support passing multiple nodes here?
How useful is it to have all content concatenated. There won't be an easy way to distinguish where one ends and the next starts, correct?

Reply:
thanks for catching this - didn't realize there was `DATUM`

Reply:
You can now also get rid of the for loop